build.gradle

```diff
@@ -53,16 +62,19 @@ dependencies {
 	annotationProcessor "org.ow2.asm:asm-tree:${project.asm_version}"
 	annotationProcessor "org.ow2.asm:asm-util:${project.asm_version}"
 
-	implementation('net.fabricmc:sponge-mixin:0.9.4+mixin.0.8.2') {
+	required('net.fabricmc:sponge-mixin:0.9.4+mixin.0.8.2') {
 		exclude module: 'launchwrapper'
 		exclude module: 'guava'
 	}
-	implementation 'net.fabricmc:tiny-mappings-parser:0.2.2.14'
-	implementation 'net.fabricmc:tiny-remapper:0.4.2'
-	implementation 'net.fabricmc:access-widener:1.0.0'
+	required 'net.fabricmc:tiny-mappings-parser:0.2.2.14'
+	required 'net.fabricmc:tiny-remapper:0.4.2'
+	required 'net.fabricmc:access-widener:1.0.0'
+
+	required 'com.google.jimfs:jimfs:1.2-fabric'
+	// TODO REMOVE ME!! by shading into jimfs
+	required 'com.google.guava:guava:30.1.1-jre'
```

Comment:
I will setup the building of jimfs with guava shaded/repackaged in.

Reply:
And you can probably consider proguard shrink the shaded guava